crates/pallet-runtime-configs/src/lib.rs

```diff
@@ -0,0 +1,77 @@
+// Copyright (C) 2022 Subspace Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Pallet for tweaking the runtime configs for multiple network.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+pub use pallet::*;
+
+#[frame_support::pallet]
+mod pallet {
+    use frame_support::pallet_prelude::*;
+    use frame_system::pallet_prelude::*;
+
+    #[pallet::pallet]
+    #[pallet::generate_store(pub trait Store)]
+    pub struct Pallet<T>(_);
+
+    /// Enable the signed extension `DisablePallets`.
+    #[pallet::storage]
+    #[pallet::getter(fn enable_disable_pallets)]
+    pub type EnableDisablePallets<T> = StorageValue<_, bool, ValueQuery>;
+
+    #[pallet::config]
+    pub trait Config: frame_system::Config {}
+
+    #[pallet::call]
+    impl<T: Config> Pallet<T> {
+        /// Set new value of [`EnableDisablePallets`].
+        #[pallet::weight(T::DbWeight::get().writes(1))]
+        pub fn set_enable_disable_pallets(origin: OriginFor<T>, new: bool) -> DispatchResult {
```

Comment:
Shouldn't we just need to enable it once? I don't see why would we want to set it back and forth on the fly, should be just a fuse, we have a few of those in `pallet-subspace` already.